codegen/tests/simplegrpc/grpc_serviceinterface.go

```diff
@@ -4,7 +4,7 @@ package simplegrpc
 import (
```

Comment:
This file should be deleted.

Reply:
Awesome, thanks will do!

Reply:
Fixed in https://github.com/anz-bank/sysl-go/pull/14/commits/70cf7698bd11574944d9899c603cf44353ab29c7